packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -147,8 +186,25 @@ export default {
   border: 1px solid green;
 }
```

Comment:
I'm using the above style for development but will be removed in a future PR when the components different states are styled